fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -559,6 +559,16 @@ private void processChangeRefreshScheme(RefreshSchemeDesc refreshSchemeDesc, Mat
                 }
                 asyncRefreshContext.setStep(step.getLongValue());
                 asyncRefreshContext.setTimeUnit(intervalLiteral.getUnitIdentifier().getDescription());
+            } else {
+                if (materializedView.getBaseTableInfos().stream().anyMatch(tableInfo -> {
```

Comment:
how about extract this check into the `MaterializedView` ?

Reply:
+1，  could be a method

Reply:
getTable  method  has been implemented in BaseTableInfo